less/forum/Composer.less

```diff
@@ -120,6 +120,12 @@
       max-height: 100%;
       padding-top: @header-height-phone;
 
+      // Fixes a bug where fixed position doesn't properly work in Safari mobile
+      // https://github.com/flarum/core/issues/2652
+      .mobile-safari & {
```

Comment:
Isn't this class on App, not Composer?

Reply:
yes and it is evaluated as such `.mobile-safari .Composer:not(.minimized)`

Reply:
Ah I got my ampesand positions confused